pkg/resources/role_ownership_grant.go

```diff
@@ -40,6 +40,16 @@ var roleOwnershipGrantSchema = map[string]*schema.Schema{
 			"REVOKE",
 		}, true),
 	},
+	"revert_ownership_to": {
```

Comment:
for consistency perhaps this should be renamed to:
`revert_ownership_to_role_name`
alternatively:
`default_owner_role_name`